devicon.json

```diff
@@ -3159,6 +3521,9 @@
     },
     {
       "name": "labview",
+      "altnames": [
+          ""
```

Comment:
Empty string altnames should be forbidden?

Reply:
Agreed